server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailQueryMethodContract.scala

```diff
@@ -243,7 +242,7 @@ trait EmailQueryMethodContract {
   }
 
   @Test
-  def listMailsShouldBeIdempotent(server: GuiceJamesServer): Unit = {
+  def listMailsShouldBeSortedDESCByArrivalAtWithDefaultIsAscendingProperty(server: GuiceJamesServer): Unit = {
```

Comment:
Since you provided a comparator the list should be sorted in an ascending order and not a descending one

Reply:
```suggestion
  def listMailsShouldBeSortedInAscendingOrderByDefault(server: GuiceJamesServer): Unit = {
```